src/java/org/apache/cassandra/service/DataResolver.java

```diff
@@ -650,9 +651,10 @@ private UnfilteredPartitionIterator extendWithShortReadProtection(UnfilteredPart
 
         private boolean partitionsFetched; // whether we've seen any new partitions since iteration start or last moreContents() call
 
-        private ShortReadPartitionsProtection(InetAddress source, DataLimits.Counter singleResultCounter, DataLimits.Counter mergedResultCounter)
+        private ShortReadPartitionsProtection(InetAddress source, Runnable preFetchCallback, DataLimits.Counter singleResultCounter, DataLimits.Counter mergedResultCounter)
```

Comment:
We could break this line:
```suggestion
        private ShortReadPartitionsProtection(InetAddress source, 
                                              Runnable preFetchCallback, 
                                              DataLimits.Counter singleResultCounter, 
                                              DataLimits.Counter mergedResultCounter)
```